code/datums/wires/conveyor.dm

```diff
@@ -0,0 +1,17 @@
+/datum/wires/conveyor
+	holder_type = /obj/machinery/conveyor_switch
+	proper_name = "conveyor"
+	var/mob/fingerman
```

Comment:
Please document this variable to meet dmdoc requirements.

[More information on our documentation requirements can be found here](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md)